include/picongpu/Perf.cpp

```diff
@@ -0,0 +1,57 @@
+// perf.cc
+#include "pmacc/eventSystem/Perf.hpp"
+#include <vector>
+#include <tuple>
+#include <map>
+#include <cmath>
+
+namespace Performance {
```

Comment:
please rename the namespace `performance` see https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#2-project   (code formating in this document is outdated because we use now clang-format)

Reply:
@PrometheusPi 
> @frobnitzem Great addition and thanks for your contribution!
> Just a quick question: Is the performance output optional?

It's not optional at present.  It's fast, but rather noisy (since each MPI rank prints its data at the end-of-run).  The data output per rank is fixed at about 4.3 KB.

Many codes leave high-level performance metrics always on (see, e.g. [Gromacs "M E G A - F L O P S   A C C O U N T I N G"](https://github.com/michellab/BioSimSpace/issues/89)), which is helpful for users tweaking parallelization settings.